src/components/FilterList/SearchResults/SearchResultsVirtualList.tsx

```diff
@@ -0,0 +1,32 @@
+import { SearchResultsProps } from '@/types/types';
```

Comment:
Sort props to match linting

Also, I personally don't think that this Props component should live in the general types.d.ts file.  That file should include types that are relevant to all parts of the UI, and this type is only relevant to the `SearchResults` component. 

Create a local types folder? Or export it from the util file?  Thoughts @ylvaselling  and @engbergandreas =

Reply:
My thoughts was also that it would make sense to have a types file in this folder but @engbergandreas asked me to put it in the global types file... I think there are pros and cons with both